### PR TITLE
Add openvpn_exporter to vpn-seed-server image

### PR DESCRIPTION
**Pull request by MartinWeindel**
**What this PR does / why we need it**:
In the HA mode there is no envoy side-car in the `vpn-seed-server` pod anymore.
To collect prometheus metrics about traffic and connected clients, the openvpn_exporter is added to the `vpn-seed-server` image.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```noteworthy operator
Add openvpn_exporter to vpn-seed-server image
```

### CODE REVIEW

File: seed-server/Dockerfile [hunk 1]
```diff
@@ -12,13 +12,26 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+############# gobuilder
+FROM golang:1.19.3 AS gobuilder
+
+WORKDIR /build
+COPY ./VERSION ./VERSION
+COPY ./.git ./.git
```
Comment (ScheererJ) [inline seed-server/Dockerfile:20]: Why do we need the `.git` folder?

File: shoot-client/Dockerfile [hunk 2]
```diff
@@ -12,16 +12,17 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-############# builder
+############# gobuilder
 FROM golang:1.19.3 AS gobuilder
 
 WORKDIR /build
 COPY ./VERSION ./VERSION
+COPY ./.git ./.git
```
Comment (ScheererJ) [inline shoot-client/Dockerfile:20]: Same as above: Why do we need it?

**Review by ScheererJ — COMMENTED** [review]
Looks reasonable. Just have a simple question, though.

File: seed-server/Dockerfile (hunk 1 above)
Comment (MartinWeindel) [reply seed-server/Dockerfile:20]: Needed for calculation of the effective version stored in the go executable.

**Review by ScheererJ — APPROVED** [review]
/lgtm

**Pull request merged**